src/indexer/updater.ts

```diff
@@ -32,7 +32,7 @@ function isScriptPubKeyHexOpReturn(scriptPubKeyHex: string) {
 export type UpdaterTx = {
   txid: string;
   vin: ({ txid: string; vout: number; txinwitness: string[] } | { coinbase: string })[];
-  vout: { scriptPubKey: { hex: string; address?: string } }[];
+  vout: { scriptPubKey: { hex: string; address?: string }, value?: number }[];
```

Comment:
Note UpdaterTx is just a view of Tx from rpcclient of the fields needed, and value is a required field. Thus I'd make value required here too.